hazelcast/src/main/java/com/hazelcast/internal/config/ConfigValidator.java

```diff
@@ -43,6 +46,11 @@
 
     private static final ILogger LOGGER = Logger.getLogger(ConfigValidator.class);
 
+    private static final EnumSet<EvictionConfig.MaxSizePolicy> SUPPORTED_ON_HEAP_NEAR_CACHE_MAXSIZE_POLICIES
+            = EnumSet.of(EvictionConfig.MaxSizePolicy.ENTRY_COUNT);
+
+    private static final boolean ENTERPRISE = BuildInfoProvider.getBuildInfo().isEnterprise();
```

Comment:
@jerrinot just changed this in this PR: https://github.com/hazelcast/hazelcast/pull/10903

I think this should be reverted to the actual state, even if it's just the EE flag of the build info.